packages/frontend/src/components/buy/providers/transak.js

```diff
@@ -0,0 +1,23 @@
+import { stringifyUrl } from 'query-string';
+
+import { TRANSAK_API_KEY, TRANSAK_BUY_URL } from '../../../config';
+
+export function buildTransakPayLink(accountId) {
+    const url = {
+        url: TRANSAK_BUY_URL,
+        query: {
+            apiKey: TRANSAK_API_KEY,
+            cryptoCurrencyList: 'NEAR',
+            walletAddress: accountId,
+            // Disabled, because ay confuse users 
```

Comment:
Nit: Spelling error in comment 

Also, can you provide a bit more clarity as to why the user will be confused?

Reply:
Nevermind, I enabled this param back to ease user-flow. 
It works fine now